### PR TITLE
Updated to use Tini base image

### PR DESCRIPTION
**Pull request by michael-bryson**

### CODE REVIEW

File: release-notes-1.3.0.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
+- Updated to base the image off the [opensuse-tini-image](https://github.com/CAFapi/opensuse-tini-image)
```
Comment (dermot-hardy) [inline release-notes-1.3.0.md]: This is really just an internal re-organisation.  It doesn't have any effect on a consumer so I wouldn't update the release notes at all.  We will not release 1.3 just for this.

File: src/main/docker/Dockerfile [hunk 2]
```diff
@@ -14,26 +14,17 @@
 # limitations under the License.
 #
 
-FROM opensuse:latest
+FROM cafapi/opensuse-tini:1.0
 
 # Update the OS packages and install Java and cURL
 RUN zypper -n refresh && \
     zypper -n update && \
     zypper -n install java-1_8_0-openjdk curl
```
Comment (dermot-hardy) [inline src/main/docker/Dockerfile]: I'd rather that we moved curl into the base tini image also.  We're going to want that for node.js and everything.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: release-notes-1.3.0.md (hunk 1 above)
Comment (michael-bryson) [reply release-notes-1.3.0.md]: Reverted release notes change.

File: src/main/docker/Dockerfile (hunk 2 above)
Comment (michael-bryson) [reply src/main/docker/Dockerfile]: cURL has been removed

**Comment by michael-bryson** [comment]
Reverted the release notes change and cURL has been removed. cURL has been added to tini image and this image now references a snapshot version of Tini.

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**